kong/global.lua

```diff
@@ -168,28 +168,20 @@ function _GLOBAL.init_pdk(self, kong_config)
 end
 
 
-function _GLOBAL.init_worker_events()
+function _GLOBAL.init_worker_events(kong_config)
```

Comment:
Should check `nil` of `kong_config` just in case other uses use the old interface style?.

Reply:
I'd rather we find and fix the (very few) call sites that will need updating. The majority of the other kong global init* functions expect `kong_config`, so I'm not too worried about getting stuck in a situation where we need to call `init_worker_events()` without having initialized `kong.configuration`.